modules/oadp-1-4-0-release-notes.adoc

```diff
@@ -0,0 +1,71 @@
+
+
+[id="known-issues-1-4-0_{context}"]
+== Known issues
+
+.Backup fails when checksumAlgorithm is not set for MCG
+
+While performing a backup of any application with Noobaa as the backup location, if the `checksumAlgorithm` configuration parameter is not set, backup fails. To fix this problem, if you do not provide a value for `checksumAlgorithm` in the Backup Storage Location (BSL) configuration, an empty value is added.
+The empty value is only added for BSLs that are created using Data Protection Application (DPA) custom resource (CR), and this value is not added if BSLs are created using any other method.
+For more information, see xref:changes-oadp-1-3-to-1-4_oadp-1-4-release-notes[Changes from OADP 1.3 to 1.4].
+link:https://issues.redhat.com/browse/OADP-4274[OADP-4274]
+
+For a complete list of all known issues in this release, see the list of link:https://issues.redhat.com/issues/?filter=12438506[OADP 1.4.0 known issues] in Jira.
+
+
+[id="upgrade-notes-1-4-0_{context}"]
+== Upgrade notes
+
+[NOTE]
+====
+Always upgrade to the next minor version. *Do not* skip versions. To update to a later version, upgrade only one channel at a time. For example, to upgrade from {oadp-first} 1.1 to 1.3, upgrade first to 1.2, and then to 1.3.
+====
+
+[id="changes-oadp-1-3-to-1-4_{context}"]
+=== Changes from OADP 1.3 to 1.4
+
+The Velero server has been updated from version 1.12 to 1.14. Note that, there are no changes in the Data Protection Application (DPA).
+
+This changes the following:
+
+* The `velero-plugin-for-csi` code is now available in the Velero code, which means no `init` container is needed for the plugin anymore.
+
+* Velero changed client Burst and QPS defaults from 30 and 20 to 100 and 100 respectively.
+
+
+* The `velero-plugin-for-aws` plugin updated default value of the `spec.config.checksumAlgorithm` field in BackupStorageLocations (BSLs) from `""` (no checksum calculation) to the `CRC32` algorithm. For more information, see link:https://github.com/vmware-tanzu/velero-plugin-for-aws/blob/release-1.10/backupstoragelocation.md[Velero plugins for AWS Backup Storage Location]. The checksum algorithm types are known to work only with AWS. 
+Several S3 providers require the `md5sum` to be disabled by setting the checksum algorithm to `""`. Confirm `md5sum` algorithm support and configuration with your storage provider. 
++
+In  OADP 1.4, the default value for BSLs created within DPA for this configuration is `""`. This default value means that the `md5sum` is not checked, which is consistent with OADP 1.3. For BSLs created within DPA, update it by using the `spec.backupLocations[].velero.config.checksumAlgorithm` field in the DPA. If your BSLs are created outside DPA, you can update this configuration by using `spec.config.checksumAlgorithm` in the BSLs.
+
+[id="upgrade-steps-1-4-0_{context}"]
```

Comment:
Why is there a header with no content? Should this be in the assembly between the two modules instead? 

Additionally I recommend finding other wording, like "Upgrading from 1.3 to 1.4", as this reinforces the requirement to be on the previous version first. "Upgrading steps" sounds like the steps are being upgraded, not the OADP platform.

Reply:
@apurvabhide17 

It renders the same H3 as the section below it, and it's unclear to me how this is supposed to be interpreted, considering there are procedures below this header. 

You have one of three options: 
1. Remove this empty header from the module and put in it in the assembly level at H2, if the sections below are supposed to be nested beneath it. 
Example:
```
Upgrade Notes (H2)
Upgrading from OADP 1.3 to 1.4 (H2)
	Backing up the DPA Configuration (H3)
```

2. Remove this empty header from the module and put in it in the assembly level at H3, and change the offset to the procedures below to H4, if you want all of this content nested under the Upgrade Notes H2 section above. 
Example: 
```
Upgrade Notes (H2) 
	Upgrading from OADP 1.3 to 1.4 (H3)
		Backing up the DPA Configuration (H4)
```

3. Just remove this header from this file and let all the procedures below live under the "Upgrade Notes" H2 header. 
Example: 
```
Upgrade Notes (H2)
	Backing up the DPA Configuration (H3)
```
I will not be merging this PR unless this is fixed.  Please remember to squash your commits prior to merge.

Reply:
@sheriff-rh, I preferred your 3rd suggestion.
Could you check it now and let me know if there is any other suggestion?

Reply:
fixed